tests/lib/nsxt_functtest.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2015 The Capirca Project Authors All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Functional test class for nsxt.py."""
+
+import copy
+import optparse
+from absl.testing import absltest
+from xml.etree import ElementTree as ET
+
+from capirca.lib import naming
+from capirca.lib import nsxt
+from capirca.lib import policy
+from capirca.tests.lib import nsxt_mocktest
+
+
+class NsxtFunctionalTest(absltest.TestCase):
+  """Functional testing for NSX-T."""
+
+  def setUp(self):
+    """Call before every test case."""
+    super().setUp()
+    parser = optparse.OptionParser()
+    parser.add_option(
+        '-d',
+        '--def',
+        dest='definitions',
+        help='definitions directory',
+        default='../def')
+    (FLAGS, args) = _parser.parse_args()
+    self.defs = naming.Naming(FLAGS.definitions)
+
+  def tearDown(self):
+    super().tearDown()
+    pass
+
+  def runTest(self):
+    pass
+
+  def test_nsxt_policy(self):
+    pol = policy.ParsePolicy(nsxt_mocktest.POLICY, self.defs)
+    exp_info = 2
+    nsx = copy.deepcopy(pol)
+    fw = nsxt.Nsxt(nsx, exp_info)
+    output = str(fw)
+
+    # parse the xml
```

Comment:
This entire test seems strange. Why is it processing bits of XML if what we're generating is JSON? Did this test execute and pass for you?

Reply:
All the XML content was removed with the last commit, it was a bit of a mess moving from one repo to another.